tests/BUILD.bazel

```diff
@@ -218,9 +218,9 @@ flatbuffer_cc_library(
 
 flatbuffer_ts_library(
     name = "typescript_ts_fbs",
-    srcs = ["typescript_keywords.fbs"],
+    srcs = ["ts/typescript_keywords.fbs"],
```

Comment:
generally we place the BUILD file in the same directory as the thing we are using. So it might make sense to make a new `BUILD.bazel` in `tests/ts` that is everything from this file related to TS moved there.

Reply:
@bjornharrtell I think this is actually your issue. Looks like the bazel typescript rules end up evaluating relative to the build file dir, not the location of the ts file itself or something. Seems weird, but :shrug: .

Reply:
That led me into the right direction and seems to be working now after several trial and errors. :)